Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/fubar/ubuntu-indexer
```

Comment:
sounds right to me -- we should keep the DXR building script in the DXR repo and the deploy specifics over in the deploy repo. @klibby opinion?